php/libraries/Utility.class.inc

```diff
@@ -1246,5 +1246,50 @@ class Utility
         }
         return checkdate($dateElement['M'], $dateElement['d'], $dateElement['Y']);
     }
+
```

Comment:
Given these changes to the Utility file, let's also move your function NDB_Menu_Filter_cpg_browser:getDistinctValues() into this file